src/libraries/System.Memory/tests/ReadOnlySpan/Split.char.cs

```diff
@@ -0,0 +1,260 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class ReadOnlySpanTests
+    {
+        [Fact]
+        public static void SplitNoMatchSingleResult()
+        {
+            ReadOnlySpan<char> value = "a b";
+
+            string expected = value.ToString();
+            var enumerator = value.Split(',');
+            Assert.True(enumerator.MoveNext());
+            Assert.Equal(expected, value[enumerator.Current].ToString());
+        }
+
+        [Fact]
+        public static void DefaultSpanSplitEnumeratorBehavior()
+        {
+            var charSpanEnumerator = new SpanSplitEnumerator<string>();
```

Comment:
shouldn't this read `SpanSplitEnumerator<char>();` ?

Reply:
Yes, that's correct, should have been `char`. Don't think it's important enough to warrant an immediate fix, as the implementation does not change behaviour based on `T`, but I'll note it down to append to a future PR.

Reply:
Resolved in https://github.com/dotnet/runtime/pull/37742